packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import { useWalletMigration } from '@deriv/api';
```

Comment:
Same here, we shouldn't be using hooks from @deriv/api.

Reply:
`@deriv/hooks`'s `useWalletMigration` says use `@deriv/api`'s version
https://github.com/binary-com/deriv-app/pull/13925/files#diff-d4c30edbbf680b7ec1c7d4b47a50c2604646a2513a6764b5cc1587bee8a349b9L5-L10

Also, `@deriv/api` is already being used in `@deriv/appstore`

Reply:
Okay, apparently the "deprecated" message in `@deriv/hooks` is no longer relevant, so I've removed the hook from `@deriv/api` and now am using the `@deriv/hooks` version

Reply:
Using hooks package now